compose/cli/command.py

```diff
@@ -61,12 +61,17 @@ def perform_command(self, options, handler, command_options):
         project = self.get_project(
             self.get_config_path(explicit_config_path),
             project_name=options.get('--project-name'),
+            tls=options.get('--tls'),
+            tls_ca_cert=options.get('--tlscacert'),
+            tls_cert=options.get('--tlscert'),
+            tls_key=options.get('--tlskey'),
+            tls_verify=options.get('--tlsverify'),
```

Comment:
Instead of passing all of these around as separate kwargs I think we should use a namedtuple and pass around a single object:

```
TLSOptions = namedtuple('TLSOptions', 'enabled ca_cert cert key verify')
```